.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
 
       - uses: purescript-contrib/setup-purescript@main
         with:
-          purescript: "0.14.0-rc3"
+          purescript: "0.14.0-rc5"
           psa: "0.8.0"
```

Comment:
I don't believe we specify the `psa` version elsewhere (just saw this)

Reply:
Fixed in latest commit. Thanks